imports/plugins/included/payments-stripe/server/methods/stripe.js

```diff
@@ -183,7 +183,7 @@ Meteor.methods({
     let result;
     try {
       const refundResult = StripeApi.methods.createRefund.call({ refundDetails });
-      Logger.info(refundResult);
+      Logger.debug(refundResult);
```

Comment:
We should probably be looking at these more closely in the future in code reviews.